Update documentation (#40)
kyma-incubator/service-catalog: docs/install.md

layout: docwithnav
---

> Service Catalog requires Kubernetes at version 1.11 or higher. 

Reviewer: ```suggestion
Service Catalog requires Kubernetes version 1.11 or higher. 
```

> Starting from version 0.2.x Service Catalog makes use of [Admission Webhooks](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#what-are-admission-webhooks) 

Reviewer: ```suggestion
Starting from version 0.2.x, Service Catalog uses [Admission Webhooks](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#what-are-admission-webhooks) 
```

to manage custom resources. It uses [Additional Printer Columns](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/#additional-printer-columns) 

> so you can use `kubectl` as normal to interact with Service Catalog.

Reviewer: ```suggestion
so you can use `kubectl` to interact with Service Catalog.
```
